cpp/src/io/json/nested_json_gpu.cu

```diff
@@ -593,8 +617,16 @@ struct PdaSymbolToSymbolGroupId {
     // The relative symbol group id of the current input symbol
     constexpr auto pda_sgid_lookup_size =
       static_cast<int32_t>(sizeof(tos_sg_to_pda_sgid) / sizeof(tos_sg_to_pda_sgid[0]));
+    /*
+    // Note that delimiter cannot be any of opening(closing) brace, bracket, quote, escape,
```

Comment:
Might be good to add something like "Map delimiters to newlines, and newlines to OTHER symbol group. Note that ..."